dev/create-release/release-build.sh

```diff
@@ -176,10 +176,14 @@ if [[ "$1" == "package" ]]; then
   # Source and binary tarballs
   echo "Packaging release source tarballs"
   cp -r spark spark-$SPARK_VERSION
-  # For source release, exclude copy of binary license/notice
-  rm spark-$SPARK_VERSION/LICENSE-binary
-  rm spark-$SPARK_VERSION/NOTICE-binary
-  rm -r spark-$SPARK_VERSION/licenses-binary
+
+  # For source release in v2.4+, exclude copy of binary license/notice
+  if [[ $SPARK_VERSION > "2.4" ]]; then
```

Comment:
This seems OK, but do the `rm`s cause a problem? if they fail or print an error, add `-f`? But this is fine too.

Reply:
Yea, the script fails and stops for branch-2.3. Both is ok to me, adding `f` or checking the version.

Reply:
Meh I'd just make this change. I'll merge it.

Reply:
ok, thanks!